image/image.go

```diff
@@ -26,6 +26,9 @@ import (
 // validates the manifest pointed to by the given ref
 // or returns an error if the validation failed.
 func ValidateLayout(src, ref string) error {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
This check is already an implicit property of the walk. It is not necessary.

If you want to wrap the error, wrap the error from validate, as well.